### PR TITLE
#42 Update Readme file

### PR DESCRIPTION
**Pull request by frclba**
Evolves readme.md 

## Types of changes

What types of changes does your code introduce to <repo_name>?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the CONTRIBUTING.md doc.
- [x] I have added necessary documentation (if appropriate).
- [x] Any dependent changes have been merged and published in downstream modules.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,66 @@
...
+Este software tem o objetivo de ajudar e facilitar a vida de pessoas da área fitness. Assim podemos dar um exemplo como: Um prestador de serviços da área fitness (personal trainer ou nutricionista) se cadastra no app/site onde ele consegue aumentar a gama de pessoas que estão a procura desse tipo de profissional/serviço. 
+
+Já do outro lado uma pessoa procura por um serviço prestado na area fitness. 
+
+No app/site é será possível filtrar por nota que o profissional possui, a distância de algum lugar (casa, academia ou espaço ao ar livre), o preço da consulta entre outros, deixando todo o processo mais ágil e pratico para os dois.
```
Comment (luis-gustavo) [inline README.md:11]: "app/site". Alterar para apenas app

File: README.md [hunk 2]
```diff
@@ -1,22 +1,66 @@
 # SerFit
 
+[WIKI](https://unbarqdsw2022-1.github.io/2022_1_G5_SerFit/#/)
+
 ## Introdução
 
-Este software tem o objetivo de ajudar e facilitar a vida de pessoas da área fitness. Assim podemos dar um exemplo como: Um prestador de serviços da área fitness (personal trainer ou nutricionista) se cadastra no app/site onde ele consegue aumentar a gama de pessoas que estão a procura desse tipo de profissional/serviço. Já do outro lado uma pessoa procura por um serviço prestado na area fitness. No app/site é será possível filtrar por nota que o profissional possui, a distância de algum lugar (casa, academia ou espaço ao ar livre), o preço da consulta entre outros, deixando todo o processo mais ágil e pratico para os dois.
+Este software tem o objetivo de ajudar e facilitar a vida de pessoas da área fitness. Assim podemos dar um exemplo como: Um prestador de serviços da área fitness (personal trainer ou nutricionista) se cadastra no app/site onde ele consegue aumentar a gama de pessoas que estão a procura desse tipo de profissional/serviço.
```
Comment (luis-gustavo) [inline README.md:7]: remover "nutricionista"

File: README.md [hunk 3]
```diff
@@ -1,22 +1,66 @@
...
 Este repositório foi desenvolvido como um projeto da disciplina de [Arquitetura de Software](https://github.com/UnBArqDsw2022-1/)  2022-1 da UnB.
 
-## Tecnologia
+## Objetivos Principais
 
-### Tecnologias da Aplicação (WIP :construction: )
+Posicionar-se como uma plataforma de intermediação entre: `usuários` e
 
-A geração do site estático é realizada utilizando o [docsify](https://docsify.js.org/).
+`profissionais` do ramo da Educação física, saúde, bem estar, nutrição.
```
Comment (luis-gustavo) [inline README.md:21]: Apenas profissionais de educação física, saúde e bem estar. Remover nutrição

File: README.md [hunk 4]
```diff
@@ -1,22 +1,66 @@
...
+
+## A solução
+
+![image](docs/assets/brainstorm/mindmeister.png)
+
+
+---
+
+
+# Tecnologias da Aplicação
```
Comment (luis-gustavo) [inline README.md:38]: Adicionar:
* PostgreSQL
* Docker

**Review by SwampTG — APPROVED** [review]
Mudanças requisitadas pelo @luis-gustavo foram resolvidas.

**Pull request merged**